### ISSUE TITLE
pop-ups on top of popups

### ISSUE THREAD

**Issue by vvorski** [fairdemocracy/vilfredo-client#285]
This will drive people crazy having to click on every point to see the full text. 

Scrolling works just fine in a browser, let the page scroll!

![screen shot 2015-03-26 at 12 22 26](https://cloud.githubusercontent.com/assets/1789426/6846050/d62e30b8-d3b2-11e4-8261-3f6a689da422.png)

**Comment by pietrosperoni** [comment]
You are right, but my understanding is that we are already going to take away all the modals, so is it ok @vvorski if we close this, as those popups will all go away with the modals?

**Comment by giovanisp** [comment]
Again, wrong modal usage. Modal over Modal ! 
We need to simplify those  screens, why don't we create a sprint for that?

**Comment by pietrosperoni** [comment]
we can't even close 2.0.0RC and you want to make a sprint?

**Comment by giovanisp** [comment]
before 2.1 perhaps.
